flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/endpoint/hive/HiveServer2EndpointFactoryTest.java

```diff
@@ -79,7 +81,7 @@ public void testCreateHiveServer2Endpoint() {
                         Collections.singletonList(
                                 new HiveServer2Endpoint(
                                         service,
-                                        "localhost",
+                                        "127.0.0.1",
```

Comment:
The ip address of "localhost" might be different across machines. So this test is not deterministic.